.env.example

```diff
@@ -1 +1,9 @@
-BROWSER="none"
\ No newline at end of file
+BROWSER="none"
```

Comment:
Generally it's looking good for me, nice that you've used context, kinda cool, let me know when PR would be ready for review, for now I just left some minor comments

But yeah, generally look promising :P

Reply:
Thanks!
I am kind of busy right now so it may be ready for review at the end of the weekend or so. Sorry for the inconvenience. I will get this done as quickly as possible.